xml2kinto/id_generator.py

```diff
@@ -10,6 +10,9 @@ def create_id(data):
     This will be used to make sure that two deeply nested data structures are
     exactly equal.
     """
+    # We should ignore the enabled key.
+    if 'enabled' in data:
+        del data['enabled']
```

Comment:
isn't this mutating the passed in data? We want to keep this data, right? Just not use it for the id computation? (btw, why don't we want it or the id computation?)